lib/std/sha1.nim

```diff
@@ -7,7 +7,32 @@
 #    distribution, for details about the copyright.
 #
 
-## Note: Import ``std/sha1`` to use this module
+## **Note: Import `std/sha1` to use this module**
```

Comment:
This is a bit too much in-your-face.

How about just have `**Note:**` in bold? (It should be visible enough, and the examples below also show the correct way to import this module)

Reply:
Fixed ea6319757bfd67667f410869d387ffbd0d831294